lib/preview/link.js

```diff
@@ -36,6 +37,11 @@ var PreviewConstants = require('./constants');
 var PreviewImage = require('./image');
 var Result = require('./model').Result;
 
+// URLs matching any of these regexes will be cheched for an "embedURL" meta tag.
+var CHECK_EMBED_URL_REGEXES = [
+  /^http(s)?:\/\/drive\.google\.com\//
```

Comment:
@johncrossman Good thought - I just checked and confirmed that `embedURL` appears in Docs as well. Will amend.